app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Why you need LinearLayout here ? 
What is happen if you delete it ?

Reply:
Well, I don't really need it in this case, only if I'd like to extend activity_main and add more views and containers in future. I can safely delete it.